primitives/consensus/common/src/block_import.rs

```diff
@@ -142,13 +142,17 @@ pub struct BlockImportParams<Block: BlockT, Transaction> {
 	/// Is this block finalized already?
 	/// `true` implies instant finality.
 	pub finalized: bool,
+	/// Intermediate values that are interpreted by block importers. Each block importer,
+	/// when handled a value, removes it from the intermediate list. The final block importer
+	/// rejects block import if there are still intermediate values remain unhandled.
```

Comment:
```suggestion
	/// rejects block import if there are still intermediate values that remain unhandled.
```

(rephrasing for clarity / grammar)